docs/attribution.md

```diff
@@ -1,18 +1,22 @@
 # Attribution
 
-Attribution is required for many of our data providers. Example language is provided below, but you are responsible for researching each project to follow their license terms. More details are available on the [Data Sources](data-sources.md) page.
+Attribution is required for using Mapzen's hosted terrain service as well as many of our data providers. Example language is provided below, but you are responsible for researching each project to follow their license terms. More details are available on the [Data Sources](data-sources.md) and the [Mapzen rights](https://mapzen.com/rights) pages.
 
 **Required attribution:**
 
-> Mapzen terrain tiles contain content courtesy of the U.S. Geological Survey and U.S. National Oceanic and Atmospheric Administration.
+**Using Mapzen's hosted service:**
```

Comment:
I think we can remove the Self Hosted bit if we say (and note removal of trailing periods):

### Required attribution for Mapzen's hosted service:

`Mapzen, U.S. Geological Survey, and U.S. National Oceanic and Atmospheric Administration`

Tight on space? A more concise credit also works: `Mapzen, USGS, and NOAA`